jme3-core/src/main/resources/Common/MatDefs/Light/PBRLighting.frag

```diff
@@ -247,6 +255,16 @@ void main(){
        ao = clamp(ao, 0.0, 1.0);
     #endif
 
+    #ifdef SPECULAR_AA
+        float sigma = 0.5f;
```

Comment:
The default values are set twice, here and in the j3md.
Also, can you remove the "f" suffix? Since it is unsupported in older specs, thanks

Reply:
wow, didnt noticed "f" suffix myself. Good catch.

Reply:
The "f" suffixes have been removed